src/pages/event-attendees/event-attendees.html

```diff
@@ -31,9 +31,9 @@
     <ion-list>
       <ion-item-group *ngFor="let entry of attendeesGrouped | keys">
         <ion-item-divider light class="caps-300">{{ entry.key }}</ion-item-divider>
-        <ion-item *ngFor="let attendee of entry.value" [ngClass]="{'item-icon-left-square':attendee.checked_in}"
-                  (click)="checkIn(attendee)">
-          <ion-icon name="checkmark" item-left [hidden]="!attendee.checked_in"></ion-icon>
+        <ion-item *ngFor="let attendee of entry.value" (click)="checkIn(attendee)">
+          <button ion-button color="secondary" [hidden]="attendee.checked_in" style="text-transform: none;float: right;" small>Check In</button>
```

Comment:
@betterclever such edits can be done entirely in the app once the basic functionality becomes flawless.